ecs.php

```diff
@@ -28,6 +30,11 @@
         __DIR__ . '/config/bundles.php',
         __DIR__ . '/config/_data/generated',
         UnaryOperatorSpacesFixer::class,
+
+        // broken on PHP 8.0
+        DisallowMultiPropertyDefinitionSniff::class . '.DisallowedMultiPropertyDefinition',
```

Comment:
is this string concat intentional?

Reply:
Yes, why?

Reply:
Never seen a class-fqcn combined with a string.

What does the suffix do?

Reply:
Ah. It's a way to ignore just part of the rule in code sniffer. I don't like it much, because it's only confusing, but some people prefer to mix multiple features into 1 sniff.

It's like feature toggle.